micro-rdk/src/esp32/tls.rs

```diff
@@ -261,6 +264,56 @@ impl Esp32TLSStream {
                             let mut fd: i32 = 0;
                             esp_idf_svc::sys::esp!(esp_tls_get_conn_sockfd(*tls_context, &mut fd))
                                 .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?;
+
+                            // set socket keep alive properties as such
+                            // KEEPIDLE is set to 120 second (time before a first keepalive probe is sent)
+                            // KEEPINTVL, KEEPCNT are set to 60 and 4 respectively
+                            // Total time before an IDLE and DEAD connection is closed =  360s
+                            let var: i32 = 1;
+                            if lwip_setsockopt(
+                                fd,
+                                SOL_SOCKET as i32,
+                                SO_KEEPALIVE as i32,
+                                &var as *const i32 as *const _,
+                                std::mem::size_of::<i32>() as socklen_t,
+                            ) < 0
+                            {
+                                return Err(std::io::Error::last_os_error());
+                            }
+                            let var: i32 = 60;
+                            if lwip_setsockopt(
+                                fd,
+                                IPPROTO_TCP as i32,
+                                TCP_KEEPINTVL as i32,
+                                &var as *const i32 as *const _,
+                                std::mem::size_of::<i32>() as socklen_t,
```

Comment:
Is it possible to declare these values as `const` somewhere (ie `const KEEPALIVE_INTVL_VAL`) and reference them directly (ie on L288 `&KEEPALIVE_INVL_VAL as *const i32 as *const _,...`)

Reply:
make sense